### PR TITLE
String comparator charts

### PR DESCRIPTION
**Pull request by samnlindsay**
Improvements to string comparator charts.

### CODE REVIEW

**Comment by RossKen** [comment]
@samnlindsay I will look at properly next week, but one immediate thing is that now we have fixed the rendering issue with the docs, we can get rid of the png files 🎉

**Comment by RossKen** [comment]
Hey @samnlindsay I have pushed a couple of small changes when reviewing this:
- Rebase ontop of master
- Rerun string comparators notebook to use prettier charts (& remove pngs as they are no longer needed)
- Remove unused import flagged in lint

There are a couple more minor changes needed then I will approve:
- fix linting - it is saying about a `df` never being used
- warnings thrown by the function e.g.
image

**Review by RossKen — CHANGES_REQUESTED** [review]
See above comment

**Comment by samnlindsay** [comment]
The unused `df` isn't anything I've changed, it's to do with how we get data into duckdb:

```python
duckdb.connect()
df = pd.DataFrame(list)
duckdb.sql("select * from df").df()
```
But in this case df isn't referenced again.

File: splink/comparison_helpers.py [hunk 1]
```diff
@@ -66,7 +65,7 @@ def comparator_score_df(list, col1, col2, decimal_places=2):
     """
     duckdb.connect()
 
-    pd.DataFrame(list)
+    df = pd.DataFrame(list)
```
[the remark this answers is not in the document]
Comment (samnlindsay) [reply splink/comparison_helpers.py:68]: Maybe it was, but by my reckoning the unchanged version shouldn't work:

```python
import duckdb
import pandas as pd

duckdb.connect()
pd.DataFrame([[1,2,3],["a","b","c"]])
duckdb.sql("""select * from df""")
```

[the remark this answers is not in the document]
Comment (RossKen) [reply splink/comparison_helpers.py]: Yea I get that, but it worked in the previous version and I'm not willing to spend more time on it now coming up with a better solution. So I have reverted the change and pushed

**Review by RossKen — APPROVED** [review]

**Pull request merged**